osu.Game/Overlays/Settings/Sections/Gameplay/GeneralSettings.cs

```diff
@@ -76,7 +76,19 @@ private void load(OsuConfigManager config)
                 {
                     LabelText = "Score display mode",
                     Bindable = config.GetBindable<ScoringMode>(OsuSetting.ScoreDisplayMode)
-                }
+                },
+                new SettingsCheckbox
+                {
+                    LabelText = "Show playfield area",
```

Comment:
would probably reword to

```suggestion
                    LabelText = "Show playfield area bounds",
```

Reply:
i'd call it "playfield border"